CHANGELOG.md

```diff
@@ -2,15 +2,16 @@
 ------------------------------------------------------------------------
 - Feature: [#77] Add "Exit OpenLoco" to the main menu.
 - Fix: [#264] Option 'Export plug-in objects with saved games' is partially cut off.
+- Fix: [#299, #430] Crash due to added null-chars when manually specifying Locomotion directory.
 - Fix: [#359] Widgets tied to tools could get stuck in pressed state.
 - Fix: [#388] Re-center Options window on scale factor change.
 - Fix: [#396] Preferred owner name is not saved.
 - Fix: [#409] Incorrect refund cost when deleting signals.
 - Fix: [#412] Game crashes after a while on Great Britain & Ireland 1930.
+- Fix: [#413, #478] Crash when opening narrow gauge tab on train purchasing window
```

Comment:
I don't think we would normally include the PR number in this unless there was no related issue filed.

Reply:
Yes… the reason I added it was because the issue (#478) was technically already fixed earlier (by #413). Do you think we should go for just either one? (Which?)

Reply:
Just the issue

Reply:
```suggestion
- Fix: [#413] Crash when opening narrow gauge tab on train purchasing window
```